File: frontend/src/app/dto/attachment-comment/attachment-comment.data.dto.ts
export class AttachmentCommentDataDto{
    private _id! : number
	private _fileId! : string
	private _commentId! : string
    private _ver! : number

    public get fileId(): string {
        return this._fileId;
    }

    public set fileId(fileId: string) {
        this._fileId = fileId;
    }

    public get commentId(): string {
        return this._commentId;
    }

    public set commentId(commentId: string) {
        this._commentId = commentId;
    }
    
    public get id(): number {
        return this._id;
    }

    public set id(id: number) {
        this._id = id;
    }


    public get ver(): number {
        return this._ver;
    }

    public set ver(ver: number) {
        this._ver = ver;
    }

}

File: backend/src/main/java/com/lawencon/ticketms/dto/attachmentcomment/AttachmentCommentDataDto.java
package com.lawencon.ticketms.dto.attachmentcomment;

public class AttachmentCommentDataDto {
	private Long id;
	private Long fileId;
	private Long commentId;
	private Integer ver;
	public Long getId() {
		return id;
	}
	public void setId(Long id) {
		this.id = id;
	}
	public Long getFileId() {
		return fileId;
	}
	public void setFileId(Long fileId) {
		this.fileId = fileId;
	}
	public Long getCommentId() {
		return commentId;
	}
	public void setCommentId(Long commentId) {
		this.commentId = commentId;
	}
	public Integer getVer() {
		return ver;
	}
	public void setVer(Integer ver) {
		this.ver = ver;
	}
}
